platform-tests/src/availability/helpers/concourse.go

```diff
@@ -116,7 +117,9 @@ func buildsWithVersion(team concourse.Team, pipelineName, resourceName, resource
 	}
 
 	resourceVersions, _, resourceExists, err := team.ResourceVersions(pipelineName, resourceName, page)
-	ExpectWithOffset(2, err).NotTo(HaveOccurred())
+	if err != nil {
+		log.Println(err.Error())
+	}
 	ExpectWithOffset(2, resourceExists).To(BeTrue())
```

Comment:
Do we know what state `resourceExists` will be in when this returns an error? I'd expect the value to be undefined (and probably the zero value - false). I think this expectation (and probably the following code) should only be run if err is nil.